src/button/index.ts

```diff
@@ -64,6 +64,9 @@ export class TapButton extends Button {
 
 declare global {
   interface HTMLElementTagNameMap {
+    // TODO: investigate this
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
     "tap-button": TapButton;
```

Comment:
I can't reproduce any problem that justifies ignoring the type checking here. Can you provide more context, please?

Reply:
I tested something while writing the component. It's now redundant, so I'll remove it.